lib/gooddata/rest/phmap.rb

```diff
@@ -19,12 +19,33 @@ class Connection
         ['/gdc/datawarehouse/instances/{id}', %r{/gdc/datawarehouse/instances/[\w]+}],
         ['/gdc/datawarehouse/executions/{id}', %r{/gdc/datawarehouse/executions/[\w]+}],
 
+        # domain dataproduct
+        ['/gdc/domains/{id}/dataproducts', %r{/gdc/domains/[\w]+/dataproducts}],
+        ['/gdc/domains/{id}/dataproducts/{data_product}', %r{/gdc/domains/[\w]+/dataproducts/[\w]+}],
```

Comment:
More specific regexes must come first. In this case `%r{/gdc/domains/[\w]+/dataproducts/[\w]+}` won't match anything because `%r{/gdc/domains/[\w]+/dataproducts}` is in front of it.

Reply:
fixed